src/bridge/cockpitcpusamples.c

```diff
@@ -112,3 +112,72 @@ cockpit_cpu_samples (CockpitSamples *samples)
   g_strfreev (lines);
   g_free (contents);
 }
+
+void
+cockpit_cpu_temperature (CockpitSamples *samples)
+{
+  for (int i = 0; TRUE; i++)
+    {
+      gsize len;
```

Comment:
The length field is optional, and you're not using it, so just pass NULL instead.